.github/workflows/python-package.yml

```diff
@@ -31,7 +31,6 @@ jobs:
     - name: Lint with flake8 and black
       run: |
         flake8 . --count --statistics --show-source --ignore=E501,W503,W605,E722
-        flake8 . --count --statistics --exit-zero --max-complexity=10 --ignore=E501,W503
```

Comment:
The output of the CI looks misleading when e.g. black fails as there are plenty of errors being thrown at the moment from this line. Should be fixed before enabling complexity checks and others.